src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -117,12 +117,16 @@ public void shootAtPlayer() {
     public void ezShot(){//still needs code to stop from shooting same tile twice. Consider using code in else statement in hardShot()
```

Comment:
Yes, it's accomplished by checking whether goodShot() returns false and continually randomizing shot location until goodShot() returns true. Once it returns true it exits the do{}while() loop and fires on the square that made goodShot() return true